invalid_argument

Hi! I'm trying to build (and then publish an arch linux package for it) yur project but I run into this error when compiling from source from the github main branch:
```
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp: In function ‘bb::AstPtr b
b::parse(const string&)’:
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:23:15: error: ‘invalid_arg
ument’ was not declared in this scope
   23 |         throw invalid_argument("No tokens to parse");
      |               ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:32:15: error: ‘invalid_arg
ument’ was not declared in this scope
   32 |         throw invalid_argument("Not all tokens consumed");
      |               ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp: In function ‘bb::AstPtr b
b::parse_expression_inner(bb::TokenIter&, bb::TokenIter&, bb::AstPtr, int)’:
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:74:23: error: ‘invalid_arg
ument’ was not declared in this scope
   74 |                 throw invalid_argument("Unbalanced brackets");
      |                       ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:83:23: error: ‘invalid_arg
ument’ was not declared in this scope
   83 |                 throw invalid_argument("Missing ternary else");
      |                       ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp: In function ‘bb::AstPtr b
b::parse_primary(bb::TokenIter&, bb::TokenIter&)’:
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:126:15: error: ‘invalid_ar
gument’ was not declared in this scope
  126 |         throw invalid_argument("Expected primary token but got end of inp
ut");
      |               ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:157:19: error: ‘invalid_ar
gument’ was not declared in this scope
  157 |             throw invalid_argument("Unbalanced parentheses");
      |                   ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:169:19: error: ‘invalid_ar
gument’ was not declared in this scope
  169 |             throw invalid_argument("Expected primary token for unary pref
ix "
      |                   ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:176:11: error: ‘invalid_ar
gument’ was not declared in this scope
  176 |     throw invalid_argument("Unexpected primary token");
      |           ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp: In function ‘bb::AstPtr b
b::make_unary_op(bb::TokenType, bb::AstPtr)’:
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:264:11: error: ‘invalid_ar
gument’ was not declared in this scope
  264 |     throw invalid_argument("Unrecognized unary operator");
      |           ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp: In function ‘bb::AstPtr b
b::make_binary_op(bb::TokenType, bb::AstPtr, bb::AstPtr)’:
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:354:11: error: ‘invalid_ar
gument’ was not declared in this scope
  354 |     throw invalid_argument("Unrecognized operator");
      |           ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp: In function ‘bb::AstPtr b
b::parse(const string&)’:
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:23:15: error: ‘invalid_arg
ument’ was not declared in this scope
   23 |         throw invalid_argument("No tokens to parse");
      |               ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:32:15: error: ‘invalid_arg
ument’ was not declared in this scope
   32 |         throw invalid_argument("Not all tokens consumed");
      |               ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp: In function ‘bb::AstPtr b
b::parse_expression_inner(bb::TokenIter&, bb::TokenIter&, bb::AstPtr, int)’:
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:74:23: error: ‘invalid_arg
ument’ was not declared in this scope
   74 |                 throw invalid_argument("Unbalanced brackets");
      |                       ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:83:23: error: ‘invalid_arg
ument’ was not declared in this scope
   83 |                 throw invalid_argument("Missing ternary else");
      |                       ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp: In function ‘bb::AstPtr b
b::parse_primary(bb::TokenIter&, bb::TokenIter&)’:
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:126:15: error: ‘invalid_ar
gument’ was not declared in this scope
  126 |         throw invalid_argument("Expected primary token but got end of inp
ut");
      |               ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp: In function ‘bb::AstPtr b
b::parse(const string&)’:
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:23:15: error: ‘invalid_arg
ument’ was not declared in this scope
   23 |         throw invalid_argument("No tokens to parse");
      |               ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:157:19: error: ‘invalid_ar
gument’ was not declared in this scope
  157 |             throw invalid_argument("Unbalanced parentheses");
      |                   ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:169:19: error: ‘invalid_ar
gument’ was not declared in this scope
  169 |             throw invalid_argument("Expected primary token for unary pref
ix "
      |                   ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:176:11: error: ‘invalid_ar
gument’ was not declared in this scope
  176 |     throw invalid_argument("Unexpected primary token");
      |           ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:32:15: error: ‘invalid_arg
ument’ was not declared in this scope
   32 |         throw invalid_argument("Not all tokens consumed");
      |               ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp: In function ‘bb::AstPtr b
b::make_unary_op(bb::TokenType, bb::AstPtr)’:
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:264:11: error: ‘invalid_ar
gument’ was not declared in this scope
  264 |     throw invalid_argument("Unrecognized unary operator");
      |           ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp: In function ‘bb::AstPtr b
b::parse_expression_inner(bb::TokenIter&, bb::TokenIter&, bb::AstPtr, int)’:
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:74:23: error: ‘invalid_arg
ument’ was not declared in this scope
   74 |                 throw invalid_argument("Unbalanced brackets");
      |                       ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:83:23: error: ‘invalid_arg
ument’ was not declared in this scope
   83 |                 throw invalid_argument("Missing ternary else");
      |                       ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp: In function ‘bb::AstPtr b
b::parse_primary(bb::TokenIter&, bb::TokenIter&)’:
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:126:15: error: ‘invalid_ar
gument’ was not declared in this scope
  126 |         throw invalid_argument("Expected primary token but got end of inp
ut");
      |               ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp: In function ‘bb::AstPtr b
b::make_binary_op(bb::TokenType, bb::AstPtr, bb::AstPtr)’:
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:354:11: error: ‘invalid_ar
gument’ was not declared in this scope
  354 |     throw invalid_argument("Unrecognized operator");
      |           ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:157:19: error: ‘invalid_ar
gument’ was not declared in this scope
  157 |             throw invalid_argument("Unbalanced parentheses");
      |                   ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:169:19: error: ‘invalid_ar
gument’ was not declared in this scope
  169 |             throw invalid_argument("Expected primary token for unary pref
ix "
      |                   ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:176:11: error: ‘invalid_ar
gument’ was not declared in this scope
  176 |     throw invalid_argument("Unexpected primary token");
      |           ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp: In function ‘bb::AstPtr b
b::make_unary_op(bb::TokenType, bb::AstPtr)’:
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:264:11: error: ‘invalid_ar
gument’ was not declared in this scope
  264 |     throw invalid_argument("Unrecognized unary operator");
      |           ^~~~~~~~~~~~~~~~
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp: In function ‘bb::AstPtr b
b::make_binary_op(bb::TokenType, bb::AstPtr, bb::AstPtr)’:
/home/mads/code/arch-packages/supercollider-plugins/supercollider-bytebeat-git/sr
c/supercollider-bytebeat-git-r25.f4e25f8/src/parse.cpp:354:11: error: ‘invalid_ar
gument’ was not declared in this scope
  354 |     throw invalid_argument("Unrecognized operator");
 
 ```

Thanks for the awesome work!

--- a/src/main.cpp
+++ b/src/main.cpp
@@ -1,4 +1,5 @@
 #include <iostream>
+#include <stdexcept>
 
 #include "parse.hpp"
 

--- a/src/parse.cpp
+++ b/src/parse.cpp
@@ -1,6 +1,8 @@
 #include "parse.hpp"
 #include "lex.hpp"
 
+#include <stdexcept>
+
 namespace bb
 {
 

